src/component/bookingStep/index.jsx

```diff
@@ -14,15 +15,15 @@ const BookingStep = () => {
   const [information, setInformation] = useState();
   const [selServices, setSelServices] = useState([]);
   const [selDateTime, setSelDateTime] = useState({});
+  const [booking, setBooking] = useState({});
+  const [showModal, setShowModal] = useState(false);
 
```

Comment:
Why do we need to create another state for booking data? just use the state you already have. State is for re-render UI, not for storing data